src/server/pingserver-rs/src/admin.rs

```diff
@@ -18,7 +19,7 @@ pub struct Admin {
     config: Arc<PingserverConfig>,
     listener: TcpListener,
     poll: Poll,
-    tls_config: Option<Arc<rustls::ServerConfig>>,
+    ssl_context: Option<SslContext>,
     sessions: Slab<Session>,
     metrics: Arc<Metrics<AtomicU64, AtomicU64>>,
```

Comment:
Yes! This will be a separate PR - I have a few thoughts on a macro system for metrics that should give us some options here. I'll open a tracking issue.